cuda/test/matrix/csr_kernels.cpp

```diff
@@ -67,7 +67,14 @@ class Csr : public ::testing::Test {
     using ComplexVec = gko::matrix::Dense<std::complex<double>>;
     using ComplexMtx = gko::matrix::Csr<std::complex<double>>;
 
-    Csr() : mtx_size(532, 231), rand_engine(42) {}
+    Csr()
+#ifdef GINKGO_FAST_TESTS
+        : mtx_size(152, 231),
```

Comment:
Should we keep the same property, nrows > ncols, in these matrices?

Reply:
I'm not sure it matters here, since mostly the rows are handled independently, and having a more diverse set of test cases shouldn't hurt?

Reply:
it should not hurt anything if still gives the same residual or uses the same strategy during the computing.
When having a diverse set in fast_test, I would consider whether we should include it in normal test.

Reply:
You raise a good point in the datasets we use for testing, but I think it's outside the scope of this PR. Technically for at least some kernels, we might need to test matrices with different properties, shape, sizes as well, ... I don't think it hurts if we do it like this for now, the fast test testing ncols > nrows and normal tests nrows > ncols. Later we might have to think if it makes sense to do a general dataset improvement.

Reply:
I see.